include/cooperative_perception/utils.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef COOPERATIVE_PERCEPTION_UTILS_HPP
+#define COOPERATIVE_PERCEPTION_UTILS_HPP
+
+#include <cmath>
+
+namespace cooperative_perception::utils
+{
+
+/** Check if two floating point numbers are equal
+ *
+ * The two numbers are considered equal if their relative distance is within an epsilon interval.
+ *
+ * @param[in] first First number
+ * @param[in] second Second number
+ * @return If the two specified numbers are equal
+ */
+constexpr auto almostEqual(double first, double second) -> bool
+{
+  constexpr auto kEpsilon{ 1e-5 };
```

Comment:
This function can be removed soon. I have another implementation that is based on units of least precision (ULPs). It will checked in with a future commit.